crates/store/re_types_core/src/loggable.rs

```diff
@@ -23,115 +23,113 @@ use crate::{Archetype, ComponentBatch, LoggableBatch};
 /// which makes it possible to work with lists' worth of data in a generic fashion.
 pub trait Loggable: 'static + Send + Sync + Clone + Sized + SizeBytes {
```

Comment:
I moved around the functions to group them better (arrow2 stuff at the end). Sorry.